packages/blade/src/components/Tooltip/__tests__/Tooltip.web.test.tsx

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/require-await */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { act, fireEvent, screen } from '@testing-library/react';
+import React from 'react';
+import { Tooltip, TooltipInteractiveWrapper } from '..';
+import type { TooltipTriggerProps } from '../types';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import { Button } from '~components/Button';
+import { paymentTheme } from '~tokens/theme';
+import assertAccessible from '~src/_helpers/testing/assertAccessible.web';
+import { InfoIcon } from '~components/Icons';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+
+const waitForPosition = () => act(async () => {});
+const animationDuration = paymentTheme.motion.duration.quick;
+
+describe('<Tooltip />', () => {
+  jest.useFakeTimers();
+
+  it('should render', () => {
```

Comment:
put a SSR test to check that this cannot be open on server ever

Reply:
added